datahub-actions/src/datahub_actions/event/event_registry.py

```diff
@@ -60,7 +60,7 @@ def from_class(cls, clazz: EntityChangeEventClass) -> "EntityChangeEvent":
     @classmethod
     def from_json(cls, json_str: str) -> "EntityChangeEvent":
         json_obj = json.loads(json_str)
-        return cls.from_obj(json_obj, True)
+        return cls.from_obj(json_obj)
```

Comment:
Do you see the 'parameters' field being correctly deserialized from json back into the EntityChangeEvent?